.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 orbs:
   node: circleci/node@5.0.0
 workflows:
-  test:
+  build-and-deploy:
```

Comment:
Here's what this workflow will look like:

<img width="1055" alt="Screen Shot 2022-02-09 at 18 00 29" src="https://user-images.githubusercontent.com/630449/153316619-717ae27c-b478-4cc8-adaf-5b4a5ae631f9.png">